### PR TITLE
Cdh4u mish gunnar2

### PR DESCRIPTION
**Pull request by cdh4u**

### CODE REVIEW

File: draft-ietf-mmusic-t140-usage-data-channel.xml [hunk 1]
```diff
@@ -302,6 +297,24 @@
         </section>
       </section>
       <section anchor="sec.sdp.ex" title="Examples">
+          <t>
+            Below is an example of an m= section describing a
+            T.140 data channel, without any dcsa attributes.
+            The default maximum character transmission rate
```
Comment (gunnarhm) [inline draft-ietf-mmusic-t140-usage-data-channel.xml]: No maximum character transmission rate applies.    That is stated in the draft.

(this is different from RFC 4103 that has 30 as default.)

**Review by gunnarhm — CHANGES_REQUESTED** [review]
Fine except the words about characters per second default to 30. It is not 30, it is no limit.

File: draft-ietf-mmusic-t140-usage-data-channel.xml (hunk 1 above)
Comment (cdh4u) [reply draft-ietf-mmusic-t140-usage-data-channel.xml]: Correct. I will fix that.

**Comment by gunnarhm** [comment]
looks good.

**Pull request merged**